src/services/ListService.ts

```diff
@@ -4,19 +4,32 @@ import { ListEntity } from "../entities";
 export default class ListService {
   constructor(private repository: IRepository<ListEntity>) {}
 
-  public async createList(fields: Partial<ListEntity>): Promise<ListEntity> {
+  public async createList(
+    fields: Partial<ListEntity>,
+    isFirstList: boolean
+  ): Promise<ListEntity> {
     const { user_id, id_status } = fields;
 
-    const list = await this.repository.create({
-      user_id,
-      id_status,
-    });
-
-    return list;
+    if (!isFirstList) {
+      const list = await this.repository.create({
+        user_id,
+        id_status,
+      });
+      return list;
+    } else {
+      fields.id_status = 2;
+      const list = await this.repository.create({
+        user_id,
+        id_status,
+      });
```

Comment:
**sugestão**: Setar o `fields.id_status = 2` dessa maneira, pelo que eu to vendo, não vai alterar em nada o list, porque a constante`id_status` é diferente da `list.id_status`.. voce pode colocar qual o `id_status` direto no `create` para criar com o status que você está querendo:

```js
const list = await this.repository.create({
  user_id,
  id_status: 2
});
```

Reply:
done